src/sx-design/src/__tests__/Heading.test.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+/* global document */
+
+import * as React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import prettier from 'prettier';
+
+import Heading from '../Heading';
+import Section from '../Section';
+
+// https://reactjs.org/docs/testing-recipes.html
+
+let container: HTMLDivElement | null = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement('div');
+  document.body?.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container?.remove();
+  container = null;
+});
+
+it('renders top level by default', () => {
+  // $FlowExpectedError[incompatible-call]
+  render(<Heading>level one</Heading>, container);
+  expect(container?.innerHTML).toMatchInlineSnapshot(`"<h1>level one</h1>"`);
+});
+
+// TODO: How should this actually behave? 🤔
+// <>
+//   <Heading>level one</Heading>   -> h1
+//   <Heading>level one</Heading>   -> h2
```

Comment:
Though it is normal to have only one h1 tag, I don't think it hurts your CEO score, nor screen readers having multiple ones h1's

Reply:
True. I captured the current behavior with a snapshot.